support-frontend/package.json

```diff
@@ -207,7 +207,7 @@
     "web-vitals": "^2.0.0",
     "webpack": "^5.76.0",
     "webpack-bundle-analyzer": "^4.5.0",
-    "webpack-cli": "^4.9.2",
+    "webpack-cli": "^5.1.4",
```

Comment:
the older version gave us the error: `options has an unknown property '_assetEmittingPreviousFiles'.`